components/GoalsAndMission/GoalsAndMission.module.css

```diff
@@ -0,0 +1,3 @@
+.container {
+  background-color: #eae7e7;
```

Comment:
From the mocks, it seems every page uses this background color except the landing page. So let's set this as the default body color and adjust the landing page to compensate (can do this in a separate PR if you'd like) See #514

Reply:
No problem!  That's a quick fix so I'll take care of it here.